lesson_07/conditionals/src/lesson7.ts

```diff
@@ -37,7 +46,34 @@ export function compareStrings(a: string, b: string): number {
  * @return The letter grade ("A+", "A", "A-", "B+", etc.).
  */
 export function convertGpaToLetterGrade(gpa: number): string {
-  return "F";
+  console.log(gpa);
+  switch (
+    true // better way to do this. want to use the equivilent of a Java Map (XavierCruz5106)
```

Comment:
Interesting solution, though I wouldn't argue this is better than just using if statements (need a couple more lines of code for this construction). Nice though.

Reply:
I agree it was the first solution that came to my head. the second was to use something like a java esque map of some sort and do a lookup instead of a comparison but im not sure how maps work in typescript so ive stray away from that approach